arcgis-ios-sdk-samples/Display information/Show a popup/ShowPopupViewController.swift

```diff
@@ -51,6 +51,11 @@ class ShowPopupViewController: UIViewController, AGSGeoViewTouchDelegate, AGSPop
             if let error = result.error {
                 self.presentAlert(error: error)
             } else if !result.popups.isEmpty {
+                // Unselect the previous feature and select the newest one.
+                self.featureLayer?.clearSelection()
+                let features = result.geoElements as? [AGSFeature]
+                let selectedFeature = features?.first
+                self.featureLayer?.select(selectedFeature!)
```

Comment:
If it has to be unwrapped, maybe unwrap in the first place to avoid the overhead of 2 optionals chained together.